readme.txt

```diff
@@ -2,6 +2,10 @@
 
 == Changelog ==
 
+= [4.7.7.1] TBD =
+
+* Fix - Rollback changes introduced in commit #8f5e431 toa allow month view to render correctly.
```

Comment:
- Let's swap out `commit #8f5e431` with `4.7.7`  (easier for normal users to understand)
- Correct `toa` to `to`